src/plugins/controls/public/range_slider/embeddable/range_slider_embeddable.tsx

```diff
@@ -397,11 +365,8 @@ export class RangeSliderEmbeddable
         searchSource.setField('query', query);
       }
 
-      const {
```

Comment:
This throws if total not contained in resp. Not intended behavior so added guard for parsing response